### ISSUE TITLE
Copy/Edit Serverconfig / "Error stopping server"

### ISSUE THREAD

**Issue by david-maus** [assetto-corsa-web/accweb#7]
Hi,

short version :)

- I clicked on the copy config button on a server
- Server was duplicated and instantly started up
- stopped the server (wanted to edit it before it goes online)
- received "Error stopping server"
- can not start or stop the servers, must restart the webapp

Terminal Output:

```

time="2019-06-05T19:25:26+02:00" level=info msg="Stopping server instance..." id=1559755519
time="2019-06-05T19:25:26+02:00" level=error msg="Error stopping instance sending an interrupt, trying to kill it next" error="not supported by windows"
time="2019-06-05T19:25:26+02:00" level=error msg="Server stopped with an error" err="exit status 1"
time="2019-06-05T19:25:26+02:00" level=info msg="Server stopped" id=1559693904
time="2019-06-05T19:25:26+02:00" level=error msg="Error closing server log file" error="invalid argument"
time="2019-06-05T19:25:28+02:00" level=info msg="Stopping server instance..." id=1559755519
time="2019-06-05T19:25:28+02:00" level=error msg="Error stopping instance sending an interrupt, trying to kill it next" error="invalid argument"
time="2019-06-05T19:25:29+02:00" level=error msg="Error stopping instance sending kill signal" error="invalid argument"
```

I had the same issue yesterday on editing a started server. I dont know if this is intented to edit a running server without stopping it before saving the config?

Stopping a server manually, edit it and starting it works correctly.

Best regards

**Comment by Kugelschieber** [comment]
That's a frontend bug. If you wait a few seconds or refresh the server list, you should see the server is not running.

**Comment by Kugelschieber** [comment]
Fixed in version 1.1.
